libgraphics/CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ if (NOT vma_POPULATED)
         ${vma_SOURCE_DIR}/vk_mem_alloc.h PROPERTIES
         LANGUAGE CXX
     )
-    target_include_directories(VulkanMemoryAllocator PUBLIC ${vma_SOURCE_DIR})
+    target_include_directories(VulkanMemoryAllocator SYSTEM PUBLIC ${vma_SOURCE_DIR})
```

Comment:
It tells the compiler that this include location contains system includes, and here the purpose is to disabled warnings from VMA.

>If SYSTEM is specified, the compiler will be told the directories are meant as system include directories on some platforms. This may have effects such as suppressing warnings or skipping the contained headers in dependency calculations (see compiler documentation). Additionally, system include directories are searched after normal include directories regardless of the order specified.